src/main/java/org/sonarsource/scanner/cli/Cli.java

```diff
@@ -66,10 +61,9 @@ Cli parse(String[] args) {
 
       } else if ("-v".equals(arg) || "--version".equals(arg)) {
         displayVersionOnly = true;
-
+        
```

Comment:
I saw in your other PRs removal of such trailing spaces (good)... Now they seem to be creeping back. Issue with Eclipse formatting settings?